server/gc/src/main/java/org/apache/accumulo/gc/GarbageCollectionEnvironment.java

```diff
@@ -72,6 +72,29 @@ public interface GarbageCollectionEnvironment {
    */
   Stream<Reference> getReferences();
 
+  /**
+   * Return true if this for the RootTable, usually setup in the constructor
+   *
+   * @return true if the GCE is for the RootTable
+   */
+  boolean isRootTable();
+
+  /**
+   * Return true if this for the MetadataTable, usually setup in the constructor
+   *
+   * @return true if the GCE is for the MetadataTable
+   */
+  boolean isMetadataTable();
+
+  /**
+   * Return a list of TableIDs for which we are considering deletes. For the root table this would
+   * be the metadata table. For the metadata table, this would be all of the other tables in the
```

Comment:
```suggestion
   * be the metadata table. For the metadata table, this would be the other tables in the
```

Reply:
Addressed in c4e0ab918bf002c83d4200af67df0578aff44e73